GetJobProfiles/Models/Recipe/Parts/GraphSyncPart.cs

```diff
@@ -4,7 +4,7 @@ namespace GetJobProfiles.Models.Recipe.Parts
 {
     public class GraphSyncPart
     {
-        public GraphSyncPart(string contentType) => Text = $"[js:configuration('ContentApiPrefix') + '/{contentType.ToLowerInvariant()}/{Guid.NewGuid()}']";
+        public GraphSyncPart(string contentType) => Text = $"[js:configuration('ContentApiPrefix').toLowerCase() + '/{contentType.ToLowerInvariant()}/{Guid.NewGuid()}']";
```

Comment:
I suspect this config value invokes the c# interpreter during the import, as there won't be an optimization to catch the values.
Can we check if it does, and if so add an optimization so that the interpreter isn't invoked.
If it does get invoked, not only will it slow the import down, but runs the risk of running out of memory.

Reply:
This doesn't seem to trigger the GraphSyncHelper optimization as the constructor is only called when the GetJobProfiles utility is ran. The compiled string then gets baked into each recipes GraphSyncPart.Text property. the js:configuration will be evaluated by the shell when the recipe runs